README.md

```diff
@@ -461,9 +454,8 @@ Examples us using enviornment variables to customize the test task.
 
 ```
 $ rake TINYTDS_UNIT_DATASERVER=mydbserver
-$ rake TINYTDS_UNIT_DATASERVER=mydbserver TINYTDS_SCHEMA=sqlserver_2008
+$ rake TINYTDS_UNIT_DATASERVER=mydbserver TINYTDS_SCHEMA=sqlserver_2016
```

Comment:
Should this be 2017 since it's basically the new default min version?

Reply:
addressed with 6684afba.